packages/migrate/package.json

```diff
@@ -40,7 +40,7 @@
   },
   "peerDependencies": {
     "@prisma/generator-helper": "*",
-    "@prisma/sdk": "*"
+    "@prisma/internals": "*"
```

Comment:
This is not strictly related to this PR, but why was `@prisma/sdk` added both as a `devDep` and as a `peerDep`?

Reply:
Both were added on Oct 9, 2019
https://github.com/prisma/prisma/commit/d69cba2aa5fa304bd41aac7a6c57f519433fbbab

Not sure why *both* were added here 🤔